t/lib/Common.pm

```diff
@@ -62,12 +65,20 @@ sub skip_unless_has_secret {
     plan skip_all => 'PERL_STRIPE_TEST_API_KEY is required' unless api_key();
 }
```

Comment:
We need to keep it because `t/07-options.t` still needs it. We can't use a mock in the `idempotency_key` test https://github.com/Crowdtilt/WebService-Stripe/blob/lwp-recorder/t/07-options.t#L17